### ISSUE TITLE
Unable to import updated elasticsearch as opensearch resource

### ISSUE THREAD

**Issue by edbighead** [aiven/terraform-provider-aiven#589]
Is there a way to import elasticsearch -> opensearch updated service into the `aiven_opensearch` terraform resource?

### Steps to reprduce

The following ES service is defined in terraform
```
resource "aiven_service" "es" {
  project                 = data.terraform_remote_state.aiven.outputs.project
  cloud_name              = "aws-eu-west-2"
  termination_protection  = true
  plan                    = "hobbyist"
  service_name            = "elasticsearch"
  service_type            = "elasticsearch"
  maintenance_window_dow  = "monday"
  maintenance_window_time = "00:00:00"
  project_vpc_id          = data.terraform_remote_state.aiven.outputs.vpc
  elasticsearch_user_config {
    elasticsearch_version = "7"
  }
}
```

Service was updated from console
![image](https://user-images.githubusercontent.com/16245796/137292650-40412eb5-ead3-4d39-b9d6-ec434f87bebc.png)

Update was successful
![image](https://user-images.githubusercontent.com/16245796/137293332-73a040e4-5e4d-44fb-b66c-1b7ff2f66b28.png)
 

Executing terraform plan results in bringing back the ES version
```
Terraform will perform the following actions:

  # aiven_service.es will be updated in-place
! resource "aiven_service" "es" {
        id                      = "dev/elasticsearch"
        # (16 unchanged attributes hidden)


!       elasticsearch_user_config {
!           elasticsearch_version = "1" -> "7"
            # (2 unchanged attributes hidden)
        }
        # (1 unchanged block hidden)
    }
```

The ES resource was removed from terraform and from plan ` terraform state rm aiven_service.es` 

A new resource was added
```
resource "aiven_opensearch" "es" {
  project                 = data.terraform_remote_state.aiven.outputs.project
  cloud_name              = "aws-eu-west-2"
  termination_protection  = true
  plan                    = "hobbyist"
  service_name            = "elasticsearch"
  maintenance_window_dow  = "monday"
  maintenance_window_time = "00:00:00"
  project_vpc_id          = data.terraform_remote_state.aiven.outputs.vpc
  opensearch_user_config {
    opensearch_version = "1"
  }
}

```
 Import the updated from Elasticsearch to Opensearch resource with
```
terraform import aiven_opensearch.es dev/elasticsearch
```

**Error**
```
aiven_opensearch.es: Importing from ID "dev/elasticsearch"...

Error: cannot set `elasticsearch_user_config` : Invalid address to set: []string{"elasticsearch_user_config"};Please make sure that all Aiven services have unique service names
```

### Versions
```
> terraform version
Terraform v0.14.8
+ provider registry.terraform.io/aiven/aiven v2.2.1
```

**Comment by mhoffm-aiven** [comment]
Hey @edbighead, 

Thanks for reporting, I was able to reproduce the issue and am looking into it.

**Comment by mhoffm-aiven** [comment]
Hey @edbighead,

could you try from the latest master ( not released yet ); it works for me now.

**Comment by edbighead** [comment]
@mhoffm-aiven thanks, will test it and let you know

**Comment by edbighead** [comment]
@mhoffm-aiven I was able to successfully import updated OpenSearch resource.
```
$ terraform import aiven_opensearch.es dev/elasticsearch
aiven_opensearch.es: Importing from ID "dev/elasticsearch"...
aiven_opensearch.es: Import prepared!
  Prepared aiven_opensearch for import
aiven_opensearch.es: Refreshing state... [id=dev/elasticsearch]
```

**Comment by mhoffm-aiven** [comment]
@edbighead  thanks for verifying! I'll close this issue after the next release then!

**Comment by mhoffm-aiven** [comment]
Sorry, i slipped and closed the issue on accident.

**Comment by gbuirel** [comment]
Hello, is this fixed in the provider v2.3.0 ? 
I've tried and faced the same error

**Comment by mhoffm-aiven** [comment]
Hey @gbuirel,

no but the fix should be included in the next version!

**Comment by mhoffm-aiven** [comment]
just released 2.3.1, if it works with that version we could close i think

**Comment by edbighead** [comment]
Closing as fixed
